docs/changes/0.23.0.rst

```diff
@@ -0,0 +1,44 @@
+Changelog for QCoDeS 0.23.0
+===========================
+
+The February 2021 release of QCoDeS.
+
+---------
+Improved:
+---------
+
+- Add ``pandas`` and ``xarray`` methods to ``DataSetCache``  matching the ``DataSet`` (#2763)
+- Fix to Avoid loading data twice in dataset's ``to_xarray_dataset`` method (#2762)
+- Fix for the ``to_pandas`` method on a live dataset by not loading from disk (#2761)
```

Comment:
This is just a bug fix for 2667 so we can either merge it with that or leave it out